app/assets/stylesheets/sulFooter.scss

```diff
@@ -1,71 +1,35 @@
-$stanford-footer-height: 250px;
-$stanford-footer-md-height: 140px;
-$sul-footer-height: 80px;
-
-#su-content {
-  height: 100%;
-  padding-bottom: $stanford-footer-height + $sul-footer-height !important;
-
-  @media (min-width: 768px) {
-    padding-bottom: $stanford-footer-md-height + $sul-footer-height !important;
-  }
+$brand-color: $stanford-cardinal !default;
+$footer-color: $white !default;
+$pre-footer-background: $stanford-fog-light !default;
```

Comment:
You may need to remove the `stanford-` from this value to match what's in the palette SCSS

Reply:
Consolidated into palette and ensured everything matched.